packages/frontend/core/src/modules/db/schema/schema.ts

```diff
@@ -26,7 +26,7 @@ export const AFFiNE_WORKSPACE_DB_SCHEMA = {
     id: f.string().primaryKey().optional().default(nanoid),
     name: f.string().optional(),
     type: f.string(),
-    show: f.string().optional(),
+    show: f.enum('always-show', 'always-hide', 'hide-when-empty').optional(),
```

Comment:
if no runtime value check for enum types, the value be written or read may not the ones expected

Reply:
maybe just for type safety is enough?

Reply:
if so, you can just simply add a type to `t.string<''>()`, it won't make any difference if there is no runtime value check by `t.enum`